cmd/orb.go

```diff
@@ -1609,3 +1629,80 @@ func orbTemplate(fileContents string, projectName string, orgName string, orbNam
 
 	return x
 }
+
+func orbDiff(opts orbOptions) error {
+	colorOpt := opts.color
+	if colorOpt != "auto" && colorOpt != "always" && colorOpt != "never" {
```

Comment:
You can also use `slices.Contains` function:

```suggestion
	allowedColorOpt := []string{"auto", "always", "never"}
	if !slices.Contains(allowedColorOpt, colorOpt) {
```